module/data/item/templates/activated-effect.mjs

```diff
@@ -158,7 +234,7 @@ export default class ActivatedEffectTemplate extends SystemDataModel {
   static #migrateRanges(source) {
     if ( !("range" in source) ) return;
     source.range ??= {};
-    if ( source.range.units === null ) source.range.units = "";
+    if ( ["none", ""].includes(source.range.units) ) source.range.units = null;
```

Comment:
Is this correct? `range.units` is `nullable: false` (`StringField`s are `nullable: false` by default).

Reply:
Hmm, well the previous code was rather confusing because it was using a migration to set `null` to `""`, and then in data preparation was setting `""` to `null`, so I'm entirely sure what values we want to be using here.

Reply:
I'm not sure either. Could have been a mistake.